src/sentry/migrations/0053_remove_old_functions.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-03-11 15:29
+from __future__ import unicode_literals
+
+import six
+import re
+
+from django.db import migrations
+from django.db.models import Q
+
+from sentry.utils.query import RangeQuerySetWrapper
+from sentry.api.event_search import get_function_alias
```

Comment:
Bringing in this concerns me. Could we vendor this function into the migration? Otherwise it's possible this behavior is going to change or break in the future.